framework/PageWizard/PageWizardNavigation.tsx

```diff
@@ -47,6 +69,50 @@ export function PageWizardNavigation() {
                   </span>
                 )}
               </button>
+              {
+                /** Nav for substeps */
+                isPageWizardParentStep(step) ? (
+                  <ol className="pf-v5-c-wizard__nav-list">
```

Comment:
I'd consider rendering this bit in a sub component just to help un-nest a bunch of the conditionals and keep this section more purely JSX; you can just define it below in the same file.

Reply:
Thanks! I added a separate component called `SupstepsNav` 👍